Mathlib/Logic/Equiv/TransferInstance.lean

```diff
@@ -55,6 +56,10 @@ theorem one_def [One β] :
 #align equiv.one_def Equiv.one_def
 #align equiv.zero_def Equiv.zero_def
 
+@[to_additive]
+noncomputable instance [UnivLE.{u, v}] [One α] : One (Shrink.{v} α) :=
```

Comment:
I don't think you even need to use UnivLE in this file; `Small.{v} α` should work for this declaration, for example.
Maybe you only need to import Mathlib.Logic.Small.Defs (to reviewers: I think it's a pretty light import, and I think it's reasonable to add these instances in this file.)